doc/tutorial/energe_lstmbasedtimeseries.md

```diff
@@ -0,0 +1,283 @@
+# LSTMBasedTimeSeriesModel on SQLFlow Tutorial
+
+This is a tutorial on how to apply a Time Series Model on [energy dataset]( https://www.dropbox.com/s/pqenrr2mcvl0hk9/GEFCom2014.zip?dl=0). 
+
+The dataset is taken from the GEFCom2014 energy forecasting [competition](Tao Hong, Pierre Pinson, Shu Fan, Hamidreza Zareipour, Alberto Troccoli and Rob J. Hyndman, "Probabilistic energy forecasting: Global Energy Forecasting Competition 2014 and beyond", International Journal of Forecasting, vol.32, no.3, pp 896-913, July-September, 2016.). It consists of years of hourly electricity load data from the New England ISO and also includes hourly temperature data. We just choose part of electricity load data which date range is from 2014-10-01 to 2014-12-31 to complete our tutorial. And, we load the dataset into MySQL manually.
+
+In this notebook, we will demonstrate how to:
+
+- Prepare time series data for training an LSTM forecasting model with SQL
+  - Data scaling
+  - Reconstruct series data
+  - Split the raw data into the train set and test set
+- Train an LSTMBasedTimeSeriesModel on the dataset.
+- Predict one or more time-step ahead electricity load data, using historical load data only. 
+
+## PART 1  Prepare Data
+
+The energy data set contains two features, one is the date-time column, and the other is electricity load data. 
+
+We can have a quick peek of the raw data by running the following standard SQL statements. 
+
+```sql
+%%sqlflow
+select * from energy.raw limit 10;
+```
+
+```sql
+%%sqlflow
+select count(*) from energy.raw;
+```
+
+### PART 1.1  Scale Data
+
+Because the raw data can't be used directly to build the model, we need to reconstruct the raw data into the series data instead. Before that, we need to scale the data first into the range of (0, 1). It should be noted that data scaling must be done before the data reconstruct shown in the next step.
```

Comment:
> we need to scale the data first into the range of (0, 1

This is min-max normalization, add a link such as https://en.wikipedia.org/wiki/Feature_scaling#Rescaling_(min-max_normalization) as an explanation?

Reply:
OK ~Thanks~